modules/statics/src/networks.ts

```diff
@@ -834,6 +834,18 @@ class RuneTestNet extends Testnet implements AccountNetwork {
   explorerUrl = 'https://runescan.io/tx/?network=stagenet';
 }
 
+class Baby extends Mainnet implements AccountNetwork {
+  name = 'Babylon';
+  family = CoinFamily.BABY;
+  explorerUrl = '';
```

Comment:
We can put explorerUrl as `https://babylon.explorers.guru/`
Explorers.guru followed similar pattern for other chains, so we can safely assume mainnet explorer URL will be same.